example/TrueSDKSample/HostViewController.swift

```diff
@@ -173,11 +173,35 @@ class HostViewController: UIViewController, UITableViewDataSource, UITableViewDe
         userDataHeightConstraint.constant = CGFloat(userDataModel.count) * propertyRowHeight
         
         TCTrueSDK.sharedManager().delegate = self
```

Comment:
I know it is an old code but also why set the delegate in didSet

Reply:
Its in viewDidLoad. I did not get your qn

Reply:
ah sorry wrong place